index.html

```diff
@@ -35,8 +35,95 @@ <h2>I’m a software developer</h2>
                         <li><a href="#" class="fa fa-github-square"></a></li>
                         <li><a href="#" class="fa fa-instagram"></a></li>
                     </ul>
+            <h1>My Recent work</h1>
```

Comment:
Kindly fix this horizontal scrolling, as it makes your webpage unresponsive

![Peek 2022-12-01 23-36](https://user-images.githubusercontent.com/26736582/205165318-d8c1813c-dcb6-4f81-8bbe-f7327233b30a.gif)

Reply:
Hi there, @bienvenuushindi 
Could you please help me with this it works fine from my end?
and how to avoid this issue in the future.